h/presenters.py

```diff
@@ -62,12 +62,9 @@ def __init__(self, annotation):
     def __getattr__(self, attr):
         return getattr(self.annotation, attr)
 
-    def __getitem__(self, key):
-        return self.annotation[key]
-
```

Comment:
`__getattr__()` is a convenience so that if you've wrapped an `Annotation` in an `AnnotationHTMLPresenter` then you can still access the `Annotation`s properties, not just the `AnnotationHTMLPresenter`'s own properties, on the `AnnotationHTMLPresenter`. You don't have to keep references to both the `Annotation` and the `AnnotationHTMLPresenter` and remember which properties to get from which.

I'm fine with it being removed if you want but I'd vote to do it as a separate PR as I think it's separate from getting the feeds code to use Postgres instead of Elastic.